src/cluster/cluster-registry.js

```diff
@@ -27,6 +27,7 @@ module.exports = class ClusterRegistry extends EventEmitter {
 	 */
 	constructor( options, connectionEndpoint ) {
 		super();
+		this.setMaxListeners( 20 );
```

Comment:
`(node) warning: possible EventEmitter memory leak detected. 11 remove listeners added. Use emitter.setMaxListeners() to increase limit.`
`Trace`
    `at ClusterRegistry.addListener (events.js:239:17)`
    `at new DistributedStateRegistry (C:\Users\Alex\dev\deepstream.io\src\cluster\distributed-state-registry.js:43:33)`
    `at new SubscriptionRegistry (C:\Users\Alex\dev\deepstream.io\src\utils\subscription-registry.js:31:32)`
    `at new PresenceHandler (C:\Users\Alex\dev\deepstream.io\src\presence\presence-handler.js:15:28)`
    `at Deepstream._init (C:\Users\Alex\dev\deepstream.io\src\deepstream.io.js:314:26)`
    `at Deepstream._checkReady (C:\Users\Alex\dev\deepstream.io\src\deepstream.io.js:345:8)`
    `at Deepstream._start (C:\Users\Alex\dev\deepstream.io\src\deepstream.io.js:174:7)`
    `at g (events.js:260:16)`
    `at emitNone (events.js:67:13)`
    `at emit (events.js:166:7)`